docs/source/tutorial_formants.rst

```diff
@@ -107,6 +114,8 @@ We can now query the results using a similar set of commands as in the previous
                   c.phone.F1.column_name('F1'), # the columns enriched by praat
                   c.phone.F2.column_name('F2'),
                   c.phone.F3.column_name('F3'))
+
```

Comment:
add a comment somewhere (maybe in text after this code) explaining that the `order.by()` statements are not strictly necessary, but have been added to clean up the outputted file.